dspace-api/src/main/java/org/dspace/discovery/DiscoverResult.java

```diff
@@ -178,28 +219,35 @@ public void setSpellCheckQuery(String spellCheckQuery) {
     }
 
     public static final class DSpaceObjectHighlightResult {
-        private DSpaceObject dso;
+        private BrowsableObject dso;
         private Map<String, List<String>> highlightResults;
+        private Map<String, List<String[]>> highlightResultsWithAuthority;
```

Comment:
Variable isn't used, neither getters nor setters are called. Can you remove it ?

Reply:
removed, see https://github.com/4Science/DSpace/pull/65/commits/cc11f4955677eaedfae70c54eb08e940bb6067fe